shared/common-adapters/dumb.native.js

```diff
@@ -21,13 +21,13 @@ const tabBarButtonMap: DumbComponentMap<TabBarButton> = {
 const tabBarBaseMock = {
   style: {flex: 1},
   children: [
-    (<TabBar.Item label='One' selected onPress={() => {}}>
+    (<TabBar.Item label='One' selected onClick={() => {}}>
```

Comment:
I got rid of TabBar.Item because it wasn't flow checking properly.

See: https://github.com/keybase/client/pull/2782/commits/790ec66a385149009f9d8f602b253142996db8c6